consensus/src/execution_ctx.rs

```diff
@@ -322,19 +320,15 @@ impl<'a, DB: Database, T: Operations + 'static> ExecutionCtx<'a, DB, T> {
         match valid {
             Ok(_) => {
                 // Re-publish the returned message
-                self.outbound.send(msg.clone()).await.unwrap_or_else(|err| {
-                    error!("unable to re-publish a handled msg {:?}", err)
-                });
+                self.outbound.try_send(msg.clone());
             }
             // This is a message from future round or step.
             // Save it in future_msgs to be processed when we reach
             // same round/step.
             Err(ConsensusError::FutureEvent) => {
                 trace!("future msg {:?}", msg);
 
-                self.outbound.send(msg.clone()).await.unwrap_or_else(|err| {
-                    error!("unable to re-publish a handled msg {:?}", err)
-                });
+                self.outbound.try_send(msg.clone());
```

Comment:
Wouldn't be useful to keep the error log?

Reply:
It's pointless here. The try_send can fail due to `Full or Closed error` and they are handled and reported in `try_send` already.

Example error:
`queue ({consensus_outbound}) is full, cap: 5000`